<!DOCTYPE UI>
<ui version="4.0" >
 <class>DVDProgress</class>
 <widget class="QDialog" name="DVDProgress" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="cursor" >
   <cursorShape>WaitCursor</cursorShape>
  </property>
  <property name="windowTitle" >
   <string>DVD Analyze</string>
  </property>
  <layout class="QGridLayout" >
   <item row="1" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="lblTotal" >
       <property name="cursor" >
        <cursorShape>WaitCursor</cursorShape>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pbTotal" >
       <property name="cursor" >
        <cursorShape>WaitCursor</cursorShape>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
       <property name="textVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="lblTitle" >
       <property name="cursor" >
        <cursorShape>WaitCursor</cursorShape>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pbTitle" >
       <property name="cursor" >
        <cursorShape>WaitCursor</cursorShape>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
       <property name="textVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblStatus" >
     <property name="cursor" >
      <cursorShape>WaitCursor</cursorShape>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
